src/main/resources/docs/installation.md

```diff
@@ -1,51 +1,80 @@
 - [Server Requirements](#server-requirements)
-    - [Development](#development)
-    - [Production](#production)
+  - [Development](#development)
+  - [Production](#production)
+  - [Setup](#setup)
 - [Installation](#installation)
-    - [Creating New Project](#creating-new-project)
-    - [Configuring](#configuring)
-    - [Serving](#serving)
-        - [Locally](#serving-locally)
-        - [Externally](#serving-externally)
+  - [Creating New Project](#creating-new-project)
+  - [Configuring](#configuring)
+  - [Serving](#serving)
+    - [Locally](#serving-locally)
+    - [Externally](#serving-externally)
+
+<a name=“server-requirements”></a>
 
-<a name="server-requirements"></a>
 ### [Server Requirements](#server-requirements)
 
 Alpas has only few system requirements. The requirements vary based on whether you want to set up a machine for
-development or for production. 
- 
-<a name="development"></a>
+development or for production.
+
+<a name=“development”></a>
+
 #### [Development](#development)
 
-<div class="sublist">
+<div class=“sublist”>
 
-* JDK >= 9.0
-* Gradle >= 5.6
+- JDK >= 9.0
+- Gradle >= 5.6
 
-> /info/ <span>You need a GitHub account to create your project based on [Alpas's project template][template].
-Once created, you can host your project anywhere you would like.</span>
+> /info/ <span>You need a GitHub account to create your project based on [Alpas’s project template][template].
+> Once created, you can host your project anywhere you would like.</span>
 
 </div>
 
-<a name="production"></a>
+<a name=“production”></a>
+
 #### [Production](#production)
 
 If you have created a [fat JAR ](https://stackoverflow.com/questions/19150811/what-is-a-fat-jar) file for serving your
 app, the only requirement is the Java Runtime `jre`. If you intend to run some Alpas console commands, such as
 `alpas migrate`, then you can set up your prod machine as if it was your [dev machine](#development).
-  
+
 > /tip/ <span>We highly recommend using [sdkman](https://sdkman.io/) for managing different versions of
-Alpas's system requirements.</span>
+> Alpas’s system requirements.</span>
+
+<a name=“setup”></a>
+
+#### [Setup](#setup)
+
+For Mac users, you can setup via brew or sdkman, both are covered below:
+
+## [Using Brew]
+
+Run the following commands in the sequence listed below:
+
+- Type `brew update` to update your brew and prepare for the needed installations.
+- To install the latest version of Gradle type `brew install gradle`
+- To install the latest version of Java type `brew cask install java`
+
+## [Using sdkman]
+
+Run the following commands in the sequence listed below:
+
+- `curl -s “https://get.sdkman.io” | bash` to install sdkman on your mac.
```

Comment:
There is an extra step missing here - either to source the sdkman script or to restart the terminal.